src/editors/data/redux/thunkActions/app.js

```diff
@@ -35,6 +41,7 @@ export const initialize = (data) => (dispatch) => {
   dispatch(module.fetchBlock());
   dispatch(module.fetchUnit());
   dispatch(module.fetchStudioView());
+  dispatch(module.fetchImages());
```

Comment:
This is great! Makes much more sense to fetch at start rather than when we open up gallery.